.github/workflows/test.yml

```diff
@@ -0,0 +1,17 @@
+name: "build-test"
+on: # rebuild any PRs and main branch changes
+  pull_request:
+  push:
+    branches:
+      - master
+      - 'releases/*'
+
+jobs:
```

Comment:
why didn't this workflow run for this PR? 👀

Reply:
since its from a forked repo.